packages/lesswrong/lib/collections/posts/schema.ts

```diff
@@ -519,7 +519,7 @@ const schema = {
         'postId', post._id
       );
       if (tagRels?.length) {
-        return Users.restrictViewableFields(currentUser, TagRels, tagRels)[0]
+        return accessFilterMultiple(currentUser, TagRels, tagRels)[0]
```

Comment:
Won't ever happen because TagRels don't have any relevant permissions requirements, but if they did, this would crash in the case where there was a tagRel that couldn't be accessed. Can be fixed by moving the `accessFilterMultiple` above the `if (tagRel?.length)` check.